src/api/procedures/removeAssetRequirement.ts

```diff
@@ -37,7 +37,7 @@ export async function prepareRemoveAssetRequirement(
 
   const { requirements } = await query.complianceManager.assetCompliances(rawTicker);
 
-  if (requirements.filter(({ id: rawId }) => u32ToBigNumber(rawId).eq(reqId)).length) {
+  if (!requirements.filter(({ id: rawId }) => u32ToBigNumber(rawId).eq(reqId)).length) {
```

Comment:
`find` makes more sense here

Reply:
Since we don't actually need the value, I guess `some` fits better here